src/System.Data.Common/tests/System.Data.Common.Tests.csproj

```diff
@@ -124,4 +124,7 @@
   <ItemGroup>
     <EmbeddedResource Include="Resources\$(AssemblyName).rd.xml" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetsNetFx)' == 'true'">
+    <Reference Include="mscorlib" />
```

Comment:
This is incorrect, we add reference to tests projects automatically, we should never be manually adding a Reference in the test project.

Reply:
you're right this wasn't needed. Removing now